src/game/game.hpp

```diff
@@ -98,6 +99,8 @@ class Game {
 
 	void forceRemoveCondition(uint32_t creatureId, ConditionType_t type, ConditionId_t conditionId);
 
+	void getCyclopediaStatistics();
```

Comment:
I recommend renaming the function from getCyclopediaStatistics to logCyclopediaStats. The current name suggests that the function might return statistics, which could lead to confusion since it actually performs logging. Using log instead of get clarifies the primary action of the function, which is to log information to the system's logger. The term CyclopediaStats succinctly captures the scope of the statistics being logged, maintaining brevity and precision. This renaming would enhance readability and better convey the function's purpose, aligning it with clear coding practices.